episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+camera or phone will be RGB. This being said, they are less common in microscopy 
+- although there are certain research fields and types of microscope that 
+commonly use RGB. A key example is imaging of tissue sections for histology or 
+pathology. You will also often use RGB images when preparing figures for papers 
+and presentations - RGB images can be opened in all imaging software (not just 
+scientific research focused ones), so are useful when preparing images for 
+display. As usual, always make sure you keep a copy of your original raw data! 
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+## Reading RGB in Napari
+
+How does Napari know an image is RGB? If an image's final dimension has a length 
+of 3 or 4, Napari will assume it is RGB and [display it as such
+](https://napari.org/stable/howtos/layers/image.html#viewing-rgb-vs-luminance-grayscale-images). 
+If loading an image from the console, you can also manually set it to load 
+as rgb:
+
+```python
+from skimage import data
+viewer.add_image(data.astronaut(), rgb=True)
+```
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## RGB histograms
+
+Make a histogram of the Skin (RGB) image using `napari-matplotlib` .
+
+- How does it differ from the image histograms we looked at last episode?
+
+:::::::::::::::::::::::: solution 
+ 
+![](fig/rgb-histogram.png){alt="RGB histogram of the Napari Skin sample image"}
+
+This histogram shows separate lines for R, G and B - this is because each 
+displayed pixel is represented by three values (R, G, B). This differs from the 
+histograms we looked at previously, where there was only one line as each 
+displayed pixel was only represented by one value.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Understanding RGB colours
+
+What colour would you expect the following (R, G, B) values to produce? Each 
+value has a minimum of 0 and a maximum of 255.
+
+- (0, 0, 255)
+- (255, 255, 255)
+- (0, 0, 0)
+- (255, 255, 0)
+- (100, 100, 100)
+
+:::::::::::::::::::::::: solution 
+ 
+![](fig/rgb-exercise.png){alt="Diagram of (R, G, B) values next to corresponding 
+colours" width='50%'}
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: keypoints 
+
+- Microscopy images can have many dimensions, usually representing time (t),
+channels (c), and spatial axes (z, y, x)
+- Napari can open images with any number of dimensions
+- Napari (and python in general) has no firm rules for axis order. Different 
+libraries and plugins will often use different conventions.
+- RGB images always have 3 channels - red (R), green (G) and blue (B). These 
+channels aren't separated into different image layers - they're instead combined 
+together to give the final image display.
```

Comment:
In a previous line you also mentioned 4 channels (I'm assuming an alpha component). I don't thin we really want to get into opacity here and what alpha means, but I do suggest harmonizing the previous statement with this one so people don't get confused.

Reply:
I think I only mention images with 4 dimensions in previous sections (e.g. 3D + channels), rather than 4 channels. I'll check though.